client/components/jetpack/licensing-activation-banner/style.scss

```diff
@@ -0,0 +1,45 @@
+.licensing-activation-banner {
+	padding-top: 16px;
+    padding-bottom: 16px;
+    color: var( --studio-white );
```

Comment:
The text color in design is `#DCDCDE`, although it looks like we don't have such a variable available. @ederrengifo, should we update the color to match the design?

Reply:
We need to be careful with any decision because the banner is displayed in two different enviroments: cloud.jetpack.com/pricing and wordpress.com/jetpack/connect/plans.

Reply:
I will recommend using the closer variable to that value. In our Figma library that color is called `grey 5` by the way. If that's a problem because of the two different environments, then I would recommend using white by default.